src/zenml/orchestrators/step_run_utils.py

```diff
@@ -236,6 +247,30 @@ def _try_to_get_docstring_and_source_code_from_template(
 
         return None, None
 
+    def _define_input_type(
```

Comment:
Maybe this is something that should happen server-side instead? All the information would be available there as well.